src/uu/ls/src/ls.rs

```diff
@@ -340,6 +349,33 @@ struct PaddingCollection {
 }
 
 impl Config {
+    // From sort.rs
```

Comment:
If this is used in both `sort` and `ls`, let's move it to uucore!

Reply:
We now do this pretty differently, but noted.

Reply:
Ah okay. Just for my understanding then: is it fundamentally different functionality or should sort also behave this way?